test/hotspot/jtreg/runtime/cds/serviceability/ReplaceCriticalClassesForSubgraphs.java

```diff
@@ -27,6 +27,7 @@
  * @summary Tests how CDS works when critical library classes are replaced with JVMTI ClassFileLoadHook
  * @library /test/lib
  * @requires vm.cds.write.archived.java.heap
+ * @requires !vm.jvmci.enabled
```

Comment:
Please add a comment to https://bugs.openjdk.org/browse/JDK-8266329 to remove this guard as part of resolving the issue.